WMF Framework/Localization.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+
+fileprivate var dictionaryRegex: NSRegularExpression? = {
```

Comment:
Yeah - it's already lazy because it's a global variable. When I wrote it up, I added `lazy` but the compiler doesn't allow it `'lazy' may not be used on an already-lazy global`.